paper/paper.md

```diff
@@ -1,124 +1,239 @@
-and precipitable water data sources [@Mikofski2017] and to determine the
-effects of spectral mismatch on different PV devices [@Mikofski2016].
-pvlib python is a foundational piece of an award, "An Open Source
-Evaluation Framework for Solar Forecasting," made under the Department
-of Energy Solar Forecasting 2 program [@DOESF2].
-
-Plans for pvlib python development includes the implementation of new
-and existing models, addition of functionality to assist with
-input/output, and improvements to API consistency.
-
-The source code for each pvlib python version is archived with Zenodo
-[@pvlibZenodo].
 
-# Acknowledgements
-
-The authors acknowledge and thank the code, documentation, and
-discussion contributors to the project.
+# Summary
 
-WH acknowledges support from the Department of Energy's Energy
-Efficiency and Renewable Energy Postdoctoral Fellowship Program
-(2014-2016), Tucson Electric Power, Arizona Public Service, and Public
-Service Company of New Mexico (2016-2018), and University of Arizona
-Institute for Energy Solutions (2017-2018).
+pvlib python is a community-developed, open-source software toolbox
+for simulating the performance of solar photovoltaic (PV) energy
+systems.  It provides peer-reviewed reference implementations of
+over 100 empirical and physics-based models from the scientific literature,
+including solar position algorithms, irradiance models, thermal models,
+and PV electrical models.  In addition to these individual low-level
+model implementations, pvlib python provides high-level constructs
+that chain these models together like building blocks to form complete
+"weather-to-power" photovoltaic system models.  It also provides functions to
+fetch and import a wide variety of weather datasets useful for PV modeling. 
+
+pvlib python has been developed since 2015 and follows modern best
+practices for open-source python software, with comprehensive automated
+testing, standards-based packaging, and semantic versioning.
+Its source code is developed openly on GitHub and releases are
+distributed via the Python Package Index (PyPI) and the conda-forge
+repository.  pvlib python's source code is made freely available under
+the permissive BSD-3 license.
+
+Here we present an update on pvlib python, describing capability
+and community development since our 2018 publication [@pvlibjoss2018].
+
+
+# Statement of need
+
+PV performance models are used throughout the field of solar phovoltaics.
+The rapid increase in scale, technological diversity, and sophistication
+of the global solar energy industry demands correspondingly more
+capable models.  Per the United States Department of Energy,
+"the importance of accurate modeling is hard to overstate" [@seto2022].
+
+Compared with other PV modeling tools, pvlib python stands out in several
+key aspects.  One is its reusable toolbox design, providing the user a
+level of flexibility beyond that of other tools.  Rather than organizing
```

Comment:
```suggestion
level of flexibility and customization beyond that of other tools.  Rather than organizing
```

Reply:
Maybe not grammatically correct, but I think it would be nice to add customization/customizable